cloudmarker/events/azstorageaccountevent.py

```diff
@@ -0,0 +1,96 @@
+"""Microsoft storage account secure transfer event.
+
+This module defines the :class:`AzStorageAccountEvent` class that identifies
+a storage account with secure transfer enabled  not set to true .
+This plugin works on the storage account properties record found in the
+``ext`` bucket of ``storage_account_properties`` records.
+
+"""
+
+
+import logging
+
+from cloudmarker import util
+
+_log = logging.getLogger(__name__)
+
+
+class AzStorageAccountEvent:
+    """Azure storage account secure transfer enabled check event plugin."""
+
+    def __init__(self):
+        """Create an instance of :class:`AzStorageAccountEvent`."""
+
+    def eval(self, record):
+        """Evaluate Azure storage account for insecure transfer enabled status.
+
+        Arguments:
+            record (dict): A storage account record.
+
+        Yields:
+            dict: An event record representing a storage account with secure
+            transfer not enabled property.
+
+        """
+        com = record.get('com')
+        ext = record.get('ext')
+        if ext.get('record_type') != 'storage_account_properties':
+            return
+
+        sec_transf_required = ext.get('secure_transfer_required')
+
+        if sec_transf_required is False:
+            yield from _get_az_storage_account_secure_transfer_event(
+                com, ext)
+
+    def done(self):
+        """Perform cleanup work.
+
+        Currently, this method does nothing. This may change in future.
+
+        """
+
+
+def _get_az_storage_account_secure_transfer_event(com, ext):
+    """Evaluate Azure storage account property for insecure transfer enabled.
+
+    Arguments:
+        com (dict): Azure storage account record `com` bucket
+        ext (dict): Azure storage account record `ext` bucket
+        raw (dict): Azure storage account record `raw` bucket
```

Comment:
`raw` is not an argument to the method `_get_az_storage_account_secure_transfer_event`